src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,41 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && (
+          <p>
+            {' Posted by  '}
+            <UserInfo user={user} />
+          </p>
+        )}
+
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments && comments.length
```

Comment:
comments always will be true (empty array returns true)

```suggestion
      {comments.length
```